### PR TITLE
Added new logo and touched up design

### PR DESCRIPTION
**Pull request by y-arjun-y**

### CODE REVIEW

File: sections/title.py [hunk 1]
```diff
@@ -61,7 +41,7 @@ def contents():
             ),
             html.Li(
                 [
-                    'Open Philanthropy grants: ',
+                    'Open Philanthropy: ',
```
Comment (hamishhuggard) [inline sections/title.py:44]: Leave original (shorter).

File: sections/title.py [hunk 2]
```diff
@@ -153,29 +137,26 @@ def title_section():
                                         "Effective Altruism",
                                         href="https://www.effectivealtruism.org/"
                                     ),
-                                    ' is a loose collective of quantitatively-minded philanthropists and do-gooders.',
+                                    ' (EA) is a movement that uses reason and evidence to do the most good.',
```
Comment (hamishhuggard) [inline sections/title.py:140]: Nice.

File: sections/title.py [hunk 3]
```diff
@@ -75,7 +55,7 @@ def contents():
             ),
             html.Li(
                 [
-                    'Rethink Priorities survey results: ',
+                    'Rethink Priorities Survey: ',
```
Comment (hamishhuggard) [inline sections/title.py:58]: Come to think of it, we should just call it "EA Survey:"

File: sections/title.py [hunk 4]
```diff
@@ -153,29 +137,26 @@ def title_section():
...
-                            ),
-                            html.P(
-                                'This website aggregates and visualizes that data.',
+                                'This website aggregates and visualizes data from several EA organisations!',
                             ),
                             contents(),
                             html.P(
                                 [
-                                    'Source code is available on ',
+                                    'All of the source code is available on ',
```
Comment (hamishhuggard) [inline sections/title.py:149]: Leave original (shorter)

File: sections/title.py [hunk 5]
```diff
@@ -153,29 +137,26 @@ def title_section():
...
+                                        "GitHub",
                                         href="https://github.com/hamishhuggard/ea_data_viz"
                                     ),
                                     '.',
                                 ]
                             ),
                             html.P(
                                 [
-                                    'Please send feedback to ',
+                                    'Got any feedback? Please send it to ',
```
Comment (hamishhuggard) [inline sections/title.py:159]: Leave original (shorter).

File: sections/title.py [hunk 6]
```diff
@@ -188,8 +169,11 @@ def title_section():
                             'text-align': 'left',
                         },
                     ),
+                    html.Br(),
+                    html.Br(),
                     html.H3(
-                        '⬇️  Scroll down. ⬇️ '
+                        'Scroll down.',
```
Comment (hamishhuggard) [inline sections/title.py:175]: Leave original (emojis helpful for user)

File: sections/title.py [hunk 7]
```diff
@@ -20,37 +20,17 @@ def heading():
...
-                        className = 'data',
-                    ),
-                    html.Span(
-                        '.com',
-                        className = 'dot-com',
-                    ),
-                ],
-                className='data-dot-com',
-            ),
+                        'Effective Altruism Data'
```
Comment (hamishhuggard) [inline sections/title.py:23]: I know the vertical alignment is bad right now, but I don't want to lose the different fonts. Leave the original for now and I'll fix it.

File: sections/title.py (hunk 4 above)
Comment (y-arjun-y) [reply sections/title.py:149]: What about the 'There are several EA organisations which collect data on grants, donors, and pledges.' part?

File: sections/title.py (hunk 3 above)
Comment (y-arjun-y) [reply sections/title.py:58]: Should I re-write it as 'EA Survey' or keep it the same?

**Pull request merged**